### PR TITLE
Adding Track2 CodeGened Compute, Network and ResourceManager RPs

### PR DESCRIPTION
**Pull request by allenjzhang**
Based on latest codegen.
Build: Passing
Tests: TBA

### CODE REVIEW

File: eng/Directory.Build.Data.props [hunk 1]
```diff
@@ -55,6 +55,7 @@
       ($NoWarn);
```
Comment (pakrym) [inline eng/Directory.Build.Data.props:55]: this line looks wrong

Comment (allenjzhang) [reply eng/Directory.Build.Data.props:55]: I know it looks duplicated with CA1308  above, but without it, build fails.

Comment (pakrym) [reply eng/Directory.Build.Data.props:55]: No, the syntax is wrong, it should be `$(NoWarn);` instead

**Review by erich-wang — APPROVED** [review]

File: sdk/compute/Azure.Management.Compute/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# README.md template
```
Comment (weshaggard) [inline sdk/compute/Azure.Management.Compute/README.md:1]: These README's are still in a bogus state, please file an issue to get them fixed before we end up shipping them this way.

**Pull request merged**